gce/sysprep/instance_setup.ps1

```diff
@@ -314,6 +274,36 @@ function Enable-RemoteDesktop {
 }
 
 
+function Configure-WinRM {
```

Comment:
I find the commands fairly confusing to read; can you make parameters more clear and add more comments?